src/ex2/pokemonClient.js

```diff
@@ -0,0 +1,43 @@
+
+class PokemonClient{
+    constructor(){}
+    async getListPokemonInfo(pokemonsIdList){
+        let pokemonsToReturn = [];
+        const allFetching = [];
+        try{
+            pokemonsIdList.forEach((pokemonId) => {
+                allFetching.push(fetch(`https://pokeapi.co/api/v2/pokemon/${pokemonId}`));
```

Comment:
better to create a function to fetch pokemon and then reuse it. something like `getPokemonById`